server/controllers/auth.js

```diff
@@ -32,7 +34,9 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
   if (user) {
     await Profile.create({
       userId: user._id,
-      name
+      name,
+      accountType: accountType ? accountType : 'petOwner'
+
```

Comment:
This is good work, ternary operators, Though I still think there is more attributes, on the profile mode